src/involvement/forms/appointment_form.py

```diff
@@ -99,3 +100,4 @@ def save(self):
             if not created:
                 appl.status = 'appointed'
                 appl.save()
+        cron.remove_old_applications()
```

Comment:
So a CRON-job is not supposed to be invoked manually, by definition. Unless there is some issue with `django-kronos` not installing the CRON-jobs correctly, I don't think there is any problem in terms of `remove_old_applications` not being called. The crontab on the server looks fine. So I think the change to invoke it without `django-kronos` should be removed.